tex/gregoriotex-signs.tex

```diff
@@ -1644,19 +1644,21 @@
 % #3 is the char of the alteration hole
 % #4 is 1 in the case of a flat for a key change, 0 otherwise
 \def\gre@alteration#1#2#3#4{%
-  % to see why we reset \gre@lastoflinecount, see comments of \GreGlyph.
+  \setbox\gre@box@temp@width=\hbox{#2}%
   \ifnum#4=0\relax %
     \gre@newglyphcommon %
+    \ifgre@firstglyph%
+      \global\gre@dimen@bolextra = \wd\gre@box@temp@width%
+      \gre@skip@temp@four = \gre@skip@alterationspace\relax%
+      \global\advance\gre@dimen@bolextra by \dimexpr\gre@skip@temp@four\relax%
+      \gre@debugmsg{bolshift}{bolextra: \the\ger@diment@bolextra}%
+%			\kern -\gre@dimen@bolextra\relax%
+    \fi%
```

Comment:
this conditional was never meant to go into TeX, it was just for the discussion. I'll make the pull request if you want, maybe it will be easier. I'll try that tomorrow.